main.py

```diff
@@ -146,15 +147,16 @@ async def request_termination_of_experiment(rid: int):
 
 
 def add_tracking_line(stmt_list: List[ast.stmt]) -> List[ast.stmt]:
-    """Modifies the given statements list to leave rtio logs that track each line.
+    """Returns the statements list created to leave rtio logs that track each line.
```

Comment:
How about: `Returns a new statement list interleaved with rtio logs for tracking each line.`